gdc_maf_tool/cli.py

```diff
@@ -41,7 +41,16 @@ def parse_args() -> argparse.Namespace:
         "--output",
         dest="output_filename",
         default="outfile.maf.gz",
-        help="Output file name for the resulting aggregate MAF (default: outfile.maf.gz).",
+        help=(
+            "Output file name for the resulting aggregate MAF (default:"
+            " outfile.maf.gz)."
+        ),
+    )
+    parser.add_argument(
+        "-d",
+        "--disable-aliquot-selection",
+        action="store_true",
+        help="Disable primary aliquot selection logic",
```

Comment:
Now that we discussed the details more regarding the selection logic for tumor vs normal, I think we should rename this feature since we are still doing aliquot selection.  How about something like `--include-all-files`?  (Feel free to suggest a better name if one comes to mind).  Then the description can be along the lines of including all maf files instead of choosing one per case.   This more accurately describes what this PR does.

Reply:
yes will change to --include-all-files, and have the short-hand flag be -a (for "all")